homework/style.css

```diff
@@ -1,3 +1,116 @@
+* {
```

Comment:
On a wide screen your UI is stretched out ovet the full width. It would be better to limit the width to, say, 960px max and center the content.

Note also that the table columns do not line up correctly vertically.

![image](https://user-images.githubusercontent.com/2788771/61955517-7fd63280-afbb-11e9-8112-a915f7ed14ab.png)